sql/information-schema.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Information Schema
+category: user guide
+---
+
+# Information Schema
+
+INFORMATION\_SCHEMA 库里面的表主要是为了兼容 MySQL 而存在，有些第三方软件会查询里面的信息。在目前 TiDB 的实现中，里面大部分只是一些空表。
+
+## 支持 Information Schema 表
+
+### CHARACTER\_SETS Table
+
+提供字符集相关的信息，其实数据是假的。TiDB 默认支持并且只支持 utf8mb4 。
+
+```sql
+mysql> select * from CHARACTER_SETS;
++--------------------|----------------------|-----------------------|--------+
+| CHARACTER_SET_NAME | DEFAULT_COLLATE_NAME | DESCRIPTION           | MAXLEN |
++--------------------|----------------------|-----------------------|--------+
+| ascii              | ascii_general_ci     | US ASCII              |      1 |
+| binary             | binary               | Binary pseudo charset |      1 |
+| latin1             | latin1_swedish_ci    | cp1252 West European  |      1 |
+| utf8               | utf8_general_ci      | UTF-8 Unicode         |      3 |
+| utf8mb4            | utf8mb4_general_ci   | UTF-8 Unicode         |      4 |
++--------------------|----------------------|-----------------------|--------+
+5 rows in set (0.00 sec)
+```
+
+### COLLATIONS Table
+
+同上。
+
+### COLLATION\_CHARACTER\_SET\_APPLICABILITY Table
+
+空表。
+
+### COLUMNS Table
+
+COLUMNS 表提供了关于所有表的列的信息。这张表里面的信息不准确，推荐使用 SHOW 语句查询：
+
+```sql
+SHOW COLUMNS FROM table_name [FROM db_name] [LIKE 'wild']
+```
+
+### COLUMN\_PRIVILEGES Table
+
+空表。
+
+### ENGINES Table
+
+ENGINES 表提供了关于存储引擎的信息。从和 MySQL 兼容性上考虑，TiDB `show engines` 的结果展示成了 InnoDB。
```

Comment:
Please check all the following content and update it according to the English version.